### DISCUSSION TITLE
Meta discussion on quanteda

### DISCUSSION THREAD

**Discussion by koheiw** [quanteda/quanteda#2205] [Announcements]
We hope that you discuss topics that do not fit to the Issues such as
  * How quanteda has been used
  * How quaneda should be developed
  * Who are using quanteda

**Comment by koheiw** [comment]
This is a nice new function 👍

**Reply by kbenoit** [reply]
Go GitHub!! 👏

**Comment by jimsharma206** [comment]
Hello! I recently passed the LSE course and I learned a lot so I want to thank you. Furthermore, I have a question. I am doing analysis on tweets, and I want to create a new column of the tokens for each tweet. Specifically, I want to add a column next to each full_text column where it is the original tweet, but tokenized.  However, since the class is a token, it won't let me create a dataframe. How should I proceed?

**Reply by koheiw** [reply]
Why don't you post the question on https://stackoverflow.com/ with a "quanteda" tag? We can answer questions there.

**Reply by kbenoit** [reply]
> Hello! I recently passed the LSE course and I learned a lot so I want to thank you. Furthermore, I have a question. I am doing analysis on tweets, and I want to create a new column of the tokens for each tweet. Specifically, I want to add a column next to each full_text column where it is the original tweet, but tokenized.  However, since the class is a token, it won't let me create a dataframe. How should I proceed? 

I agree with @koheiw, a reproducible example on SO would be better, but in short I think you could make it a tibble in which a column can be a list.
